Minio.Examples/ListBuckets.cs

```diff
@@ -31,7 +32,8 @@ static int Main()
         {
           /// Note: YOUR-ACCESSKEYID and YOUR-SECRETACCESSKEY
           /// are dummy values, please replace them with original values.
-            var client = new MinioClient("s3.amazonaws.com", "YOUR-ACCESSKEYID", "YOUR-SECRETACCESSKEY");
```

Comment:
I will do that in this PR. I wanted to see if this approach seemed OK before converting examples en masse.